aws/resource_aws_elasticache_replication_group_test.go

```diff
@@ -614,10 +614,9 @@ func TestAccAWSElasticacheReplicationGroup_NumberCacheClusters_Failover_AutoFail
 				PreConfig: func() {
 					// Simulate failover so primary is on node we are trying to delete
 					conn := testAccProvider.Meta().(*AWSClient).elasticacheconn
-					var input *elasticache.ModifyReplicationGroupInput
 
 					// Must disable automatic failover first
-					input = &elasticache.ModifyReplicationGroupInput{
+					var input *elasticache.ModifyReplicationGroupInput = &elasticache.ModifyReplicationGroupInput{
```

Comment:
Nit: Its easier to just use `input := &elasticache.ModifyReplicationGroupInput{` instead of the long form syntax